docs/platforms/flutter/troubleshooting.mdx

```diff
@@ -60,3 +60,44 @@ There is an [issue with the Sentry Flutter SDK](https://github.com/getsentry/sen
 This is an [issue](https://github.com/flutter/flutter/issues/135245) with Flutter itself and should be fixed in [Flutter 3.15](https://github.com/flutter/flutter/labels/found%20in%20release%3A%203.15).
 
 For prior versions, you can work around this by configuring the SDK only to take screenshots when the app is in the `resumed` state. To do this, set `SentryFlutterOptions.attachScreenshotOnlyWhenResumed` to `true`.
+
+## Using Flutter multi-view for Web
+
+Multi-view embedding was introduced in Flutter 3.24. In the [Flutter docs](https://docs.flutter.dev/platform-integration/web/embedding-flutter-web) you will find a detailed guide about the usage of this feature.
+
+Using Sentry in a multi-view application is possible, but there are some limitations.
+The following features do not currently support multi-view:
+
+- Screenshots via the `SentryScreenshotWidget` (which is part of the `SentryWidget`)
+- User interaction integration via the `SentryUserInteractionWidget` (which is part of the `SentryWidget`)
+- Window & Device events via the `WidgetsBindingIntegration`
+
+To prevent the `WidgetsBindingIntegration` from loading by default, you will need to remove the integration.
```

Comment:
`l`: is this necessary? the above states that `WidgetsBindingIntegration` doesn't support multi-view, but it's unclear whether any of the above features need/should be deactivated

Reply:
I suppose we need to be more specific, Screenshot and UserInteraction integrations are not added by default so users should not add/use them when enabling multiview but WidgetsBindingIntegration is enabled by default in the SDK which then requires the user to manually disable it.

@martinhaintz do you know if it's possible from the SDK side to detect whether multiview is enabled or not? If yes it makes things easy and we can just auto-disable unsupported features

Reply:
> Screenshot and UserInteraction integrations are not added by default
ah yes, that makes sense. yeah would be good to be a bit more explicit

Reply:
@buenaflor @kahest  good idea 👍 I created a PR which disables `SentryScreenshotWidget`and `SentryUserInteractionWidget` for multiview web applications. https://github.com/getsentry/sentry-dart/pull/2366